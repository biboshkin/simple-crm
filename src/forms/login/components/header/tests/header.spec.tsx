import React from "react";
import renderer from "react-test-renderer";

import { Header } from "..";

test("render correct", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
