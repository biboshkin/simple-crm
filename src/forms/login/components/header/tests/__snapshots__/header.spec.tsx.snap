// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct 1`] = `
Array [
  <div
    className="imgContainer"
  >
    <img
      alt="logo"
      className="img"
      src="logo.png"
    />
  </div>,
  <h2
    className="ant-typography title"
    style={
      Object {
        "WebkitLineClamp": null,
      }
    }
  >
    title
  </h2>,
]
`;
